src/Command/SeedCommand.php

```diff
@@ -65,6 +66,10 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             'short' => 'c',
             'help' => 'The datasource connection to use',
             'default' => 'default',
+        ])->addOption('dry-run', [
+            'short' => 'x',
```

Comment:
usually thats -d

Reply:
It’s -x for the MigrateCommand, I was copying that pattern.